quickwit/rest-api-tests/scenarii/es_compatibility/0011-exists-query.yaml

```diff
@@ -25,6 +25,16 @@ expected:
     total:
       value: 60
 ---
+json:
+  query:
+    exists:
+      field: payload
+expected:
+  hits:
+    total:
+      # one of the docs contains `"payload":{}`
```

Comment:
did you run this test against elasticsearch too locally?

Reply:
in particular is `{}` considered not having the value? Is this true in elastic and in quickwit? Both for fast field and indexed field path?

Reply:
yes, I run this test against Elastic. That's how I got the 99 value and discovered that there is a `{}` and that it is considered has not having a value in Elastic

Reply:
I thnk you are the first dev to actually run those tests against elastic too :) thx